tests/sles4sap/wizard_hana_install.pm

```diff
@@ -85,7 +85,11 @@ sub run {
     send_key 'alt-s';    # SAP SID
     send_key_until_needlematch 'sap-wizard-sid-empty', 'backspace' if check_var('DESKTOP', 'textmode');
     type_string $sid;
-    wait_screen_change { send_key 'alt-a' };    # SAP Password
+    if (is_sle('=15-SP6')) {
```

Comment:
is it possible that the shortcut also changes to "alt-p" in the subsequent versions of 15-SP6? if yes, **>=15-SP6** maybe better.

Reply:
@badboywj  Done. Please help to review again.